Tasks/PublishToAzureServiceBusV2/task.json

```diff
@@ -0,0 +1,157 @@
+{
+    "id": "BA761F24-CBD6-48CB-92F3-FC13396405B1",
+    "name": "PublishToAzureServiceBus",
+    "friendlyName": "Publish To Azure Service Bus",
+    "description": "Sends a message to Azure Service Bus using a service connection (no agent is required)",
+    "category": "Utility",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/publish-to-azure-service-bus",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=870237)",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Server"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 2,
+        "Minor": 239,
+        "Patch": 0
+    },
+    "groups": [
+        {
+            "name": "advancedProperties",
+            "displayName": "Advanced",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "connectedServiceName",
+            "aliases": [
+                "azureSubscription"
+            ],
+            "type": "connectedService:AzureRM",
+            "label": "Azure Service Bus service connection",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Select an Azure Service Bus service connection."
+        },
+        {
+            "name": "serviceBusQueueName",
+            "type": "string",
+            "label": "ms-resource:loc.input.label.serviceBusQueueName",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "ms-resource:loc.input.help.serviceBusQueueName"
+        },
+        {
+        "name": "serviceBusNamespace",
+        "type": "string",
+        "label": "ms-resource:loc.input.label.serviceBusQueueName",
```

Comment:
Copy-paste error - should be 'serviceBusNamespace', right?

Reply:
Fixed